Error in build for quote wine-git/programs/winecfg/staging.c:53:33

Hi, i try build today and get the error:
`
../wine-git/programs/winecfg/staging.c: В функции «csmt_set»:
../wine-git/programs/winecfg/staging.c:53:33: ошибка: в передаче аргумента 2 «set_reg_key»: несовместимый тип указателя [-Wincompatible-pointer-types]
   53 |         set_reg_key(config_key, "Direct3D", L"csmt", NULL);
      |                                 ^~~~~~~~~~
      |                                 |
      |                                 const char *
In file included from ../wine-git/programs/winecfg/staging.c:28:
../wine-git/programs/winecfg/winecfg.h:55:42: замечание: ожидался тип «const WCHAR *» {aka «const short unsigned int *»}, но аргумент имеет тип «const char *»
   55 | void set_reg_key(HKEY root, const WCHAR *path, const WCHAR *name, const WCHAR *value);
      |                             ~~~~~~~~~~~~~^~~~
make: *** [Makefile:275556: programs/winecfg/i386-windows/staging.o] Ошибка 1
`

I have configurated customization.conf for working with NTSync.

--- a/wine-tkg-git/wine-tkg-patches/misc/CSMT-toggle/CSMT-toggle.patch
+++ b/wine-tkg-git/wine-tkg-patches/misc/CSMT-toggle/CSMT-toggle.patch
@@ -71,7 +71,7 @@ index 00000000..6ff97674
 ++    else
 ++    {
 ++        // FALSE, we remove the csmt key letting wine use its default
-++        set_reg_key(config_key, "Direct3D", L"csmt", NULL);
+++        set_reg_key(config_key, L"Direct3D", L"csmt", NULL);
 ++    }
 + }
 + 

